apis/kueue/v1alpha1/workload_webhook.go

```diff
@@ -18,6 +18,8 @@ package v1alpha1
 
```

Comment:
Can you also add unit tests?

The integration tests are good, but they require a lot of code and they are expensive to run.

We can get most of the coverage using unit tests.

Reply:
unit tests have been added.